security/libtomcrypt/Portfile

```diff
@@ -34,10 +32,9 @@ patchfiles          patch-unbreak-10.6.diff
 
 worksrcdir          ${name}-${version}
 
-use_configure no
+use_configure       no
 
-build.args-append \
-                    CFLAGS="-DUSE_LTM -DLTM_DESC -I${prefix}/include/libtommath/"
+build.args-append   CFLAGS="-DUSE_LTM -DLTM_DESC -I${prefix}/include/libtommath/"
 build.target
 build.env           CC=${configure.cc} CFLAGS=${configure.cc_archflags}
```

Comment:
I realize this isn't related to your PR since you're just adjusting whitespace here, and we can handle it later, but this code can't work as intended. It's trying to set `CFLAGS` both as an environment var and as build args. The build arg will take precedence over the env var and we'll be left with no `-arch` flags. Looking at the build log, you can't see that because the build uses silent rules. Add `V=1` to `build.args` to disable them. Probably the makefile portgroup should be used rather than doing things manually. `-I` flags go in `CPPFLAGS`, not `CFLAGS`.

Reply:
@ryandesign Thank you, I will try that and update here.

Reply:
@ryandesign Moved cppflags to CPPFLAGS, moved archflags to CFLAGS. Seems to work fine.